packages/components/CHANGELOG.md

```diff
@@ -67,6 +67,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Fixed
 
+- `Pin Code`: `current-input` id is not set correctly
```

Comment:
From a consumer perspective I would have no clue what this means. Instead describing the focus behavior that was improved would be better.